extensions/default/src/getCustomizationModule.tsx

```diff
@@ -109,6 +109,26 @@ export default function getCustomizationModule() {
             );
           },
         },
+
+        {
+          id: 'ohif.contextMenu',
+
+          /** Applies the customizationType to all the menu items */
+          transform: function (customizationService: CustomizationService) {
+            // Don't modify the children, as those are copied by reference
```

Comment:
Does it makes sense to give an explicit reason as to why clones are being returned here? Pardon me if it should be obvious, but... 😊